distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,79 @@
+priority "grant";
+
+deny {
+  // Read, write to, or execute any file.
+  permission java.io.FilePermission "<<ALL FILES>>", "read, write, execute";
+
+  permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
+
+  // Change current security manager
+  permission java.lang.RuntimePermission "setSecurityManager";
+
+  // Modify application permissions at will.
+  permission java.security.SecurityPermission "setPolicy";
+
+  // Load classes into any protection domain
+  permission java.lang.RuntimePermission "createClassLoader";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/ddf/-" {
+  permission java.io.FilePermission "${ddf.home}${/}-", "read, write";
+  permission java.io.FilePermission "${ddf.home}", "read, write";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/org/codice/-" {
+  permission java.io.FilePermission "${ddf.home}${/}-", "read, write";
+  permission java.io.FilePermission "${ddf.home}", "read, write";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/org/eclipse/platform/-" {
+  permission java.lang.RuntimePermission "createClassLoader";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/org/apache/karaf/-" {
+  permission java.lang.RuntimePermission "createClassLoader";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant codeBase "file:/${ddf.home.policy}system/org/apache/aries/blueprint/-" {
+  permission java.lang.RuntimePermission "createClassLoader";
+};
+
+grant codeBase "file:/${ddf.home.policy}lib/-" {
+  permission java.lang.RuntimePermission "createClassLoader";
+  permission java.util.PropertyPermission "*", "read, write";
+};
+
+grant principal org.apache.karaf.jaas.boot.principal.RolePrincipal "manager", principal org.apache.karaf.jaas.boot.principal.RolePrincipal "admin" {
+  permission java.io.FilePermission "${ddf.home}${/}-", "read, write";
```

Comment:
Since this gives read write to etc does this mean if someone manages to get admin or manager access they can just edit this policy file in etc and then gain access to anywhere on the system not just ddf.home?

Reply:
We are using these roles as our "sshRole" right now, which is added back in with the next version of Karaf. The policy can be adjusted after that. It would allow you to adjust the policy file and then you could read in the new policy if the system was restarted. We are taking the additional measure of restricting ssh to the local system anyways, however. So you'd already need another account for the machine to get in.